experiments/AtmosLES/convective_bl_model.jl

```diff
@@ -283,9 +317,15 @@ function convective_bl_model(
                 moisture = moisture_bc,
                 turbconv = turbconv_bcs(turbconv)[1],
             ),
+            AtmosBC(),
```

Comment:
```suggestion
```
I'm not sure what it means to have three `AtmosBC`'s in this context. Should we delete this?